Makefile

```diff
@@ -8,6 +12,21 @@ help:
 	@echo "        Lint code with flake8, and check if black formatter should be applied."
 	@echo "    types"
 	@echo "        Check for type errors using pytype."
+	@echo "    test-actions"
+	@echo "        Run custom action unit tests"
+
+install:
+	pip install -r requirements.txt
+	python -m spacy download en_core_web_md
+	python -m spacy link en_core_web_md en 
+	pip install -e .
+
+install-dev:
+	python -m pip install --upgrade "pip<20"
```

Comment:
Ope I was going off of what we had in our CI. I guess we should either put this in install and install-dev or in neither. The pip<20 is there (I believe) to avoid issues with pip's new dependency resolver, which can take forever to install stuff